projects/Mallard/src/screens/home-screen.tsx

```diff
@@ -62,6 +63,23 @@ export const HomeScreen = ({
             })),
         [demoIssues.map(({ date }) => date)],
     )
+    const issueList = issues
+        .map(issue =>
+            useIssue<Issue | undefined>(issue)({
```

Comment:
We [can't use hooks in a loop](https://reactjs.org/docs/hooks-rules.html) so this will need some refactoring, sorry!

Why does this view need to fetch every issue? shouldn't that happen inside the issue view?